From 5052d1a2cd61bdfc086aee9d07345e11e1eae777 Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Tue, 5 Dec 2023 11:41:14 +0100
Subject: [PATCH 424/954] fix(stm32mp2): update boot API header version

Update BOOT_API_HEADER_VERSION to header version 2.2.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git b1a635729eb0543c2b76ed762c0e1d1244675fad fix(stm32mp2): update boot API header version]
Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Change-Id: I4d7290c75a7b95757936b535644556361524d12f
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/344114
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 plat/st/stm32mp2/include/boot_api.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/st/stm32mp2/include/boot_api.h b/plat/st/stm32mp2/include/boot_api.h
index d3bed76312..580a65b590 100644
--- a/plat/st/stm32mp2/include/boot_api.h
+++ b/plat/st/stm32mp2/include/boot_api.h
@@ -86,7 +86,7 @@
 /* Image Header related definitions */
 
 /* Definition of header version */
-#define BOOT_API_HEADER_VERSION					0x00020000U
+#define BOOT_API_HEADER_VERSION					0x00020200U
 
 /*
  * Magic number used to detect header in memory
-- 
2.53.0

